cpp/benchmarks/common/table_utilities.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "table_utilities.hpp"
+
+#include <cudf/transform.hpp>
+#include <cudf/reduction.hpp>
+
+int64_t estimate_size(std::unique_ptr<cudf::column> column)
+{
+  std::vector<std::unique_ptr<cudf::column>> columns;
+  columns.emplace_back(std::move(column));
+  cudf::table table{std::move(columns)};
+  return estimate_size(table.view());
+}
```

Comment:
```suggestion
int64_t estimate_size(cudf::column_view const& col)
{
  return estimate_size( cudf::table_view( {col} ) );
}
```
Just create a `table_view` from the `column_view`